status-im/status-react
# Sync public chats

Add public chats to sync. Syncs only active public chats and does not syncs messages. Does not delete public chats.

`Sync all` will synchronize public chats open on device-A on device-B, similarly to the rest of the data.

If you join a public chat on device A it will also be joined on device B. 

### Testing

1) Open chat "#status" on device A
2) Pair devices A & B
3) Click on sync-all on device A
4) Chat #status should be open on device B
5) Open chat "#test-team" on device B
6) Chat "#test-team" should open on device A
7) Open chat "#status-core" on device A
8) Chat #stauts-core should open on device B

Known limitations:

1) Chat won't be synchronized if joined when offline, will be fixed in future pr

### Features added

It will also show now devices that haven't been paired but have been detected, icons will be updated to reflect that.

Prompt for syncing: Includes https://github.com/status-im/status-react/pull/6972 so they can be tested together
Limit number of devices to 3 (the one you are using + 2 extra)


status: ready
